virtual_ecosystem/models/disturbance_testing/disturbance_testing_model.py

```diff
@@ -0,0 +1,91 @@
+"""A testing model to use in minimal ve_run testing."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from virtual_ecosystem.core.base_model import BaseDisturbance, BaseModel
+from virtual_ecosystem.core.config import Config
+from virtual_ecosystem.core.core_components import CoreComponents
+from virtual_ecosystem.core.data import Data
+from virtual_ecosystem.core.logger import LOGGER
+
+
+class DisturbanceTesting(
+    BaseDisturbance,
+    model_name="disturbance_testing",
+    model_update_bounds=("1 day", "1 year"),
+    vars_required_for_init=tuple(),
+    vars_populated_by_init=tuple(),
+    vars_required_for_update=tuple(),
+    vars_updated=tuple(),
+    vars_populated_by_first_update=tuple(),
```

Comment:
I'd probs just use `()` for an empty tuple, but it's not important

Reply:
I've used the same convention than in the testing model - copypaste and rename, so would leave it as it is.